react/features/base/connection/actions.native.ts

```diff
@@ -16,12 +19,33 @@ export * from './actions.any';
  * @returns {Function}
  */
 export function connect(id?: string, password?: string) {
-    return (dispatch: IStore['dispatch']) => dispatch(_connectInternal(id, password))
+    return (dispatch: IStore['dispatch'], getState: IStore['getState']) => {
+        const state = getState();
+        const { jwt } = state['features/base/jwt'];
+        const { iAmRecorder, iAmSipGateway } = state['features/base/config'];
```

Comment:
It's impossible for us to be a recorer or a SIP gateway, we are mobile in this code path.

Reply:
Updated.